QuickGuide/ImplementorGuide.md

```diff
@@ -0,0 +1,140 @@
+# Quickstart Guide to implementing [OGC API - Tiles](https://github.com/opengeospatial/OGC-API-Tiles)
+
+## About
+
+This guide is intended to communicate the key pieces of the Tiles API standard so that implementors can get started 
+without having to dig through the longer specification.
+
+## Introduction
+
+OGC API - Tiles provides a standard way to describe web map tile sets for clients to consume. It is the successor to 
+[WMTS](https://en.wikipedia.org/wiki/Web_Map_Tile_Service), updated to use JSON, in line with the spatial data on the web 
+best practices. It also is designed to be a 'building block', that can plug into the OGC API suite, but can also be used
+as a standalone piece. It is similar to [TileJSON](https://github.com/mapbox/tilejson-spec), adding the ability to 
+handle alternate projections and fitting into the OGC API suite of standards.
+
+
+## TileSet Link
+
+Any resource that is represented in JSON can provide a link to a ‘tileset’ JSON object that provides the necessary 
+metadata to enable a client application to formulate a tile request. This could be an OGC Collection, OGC Item (from OGC API - 
+Features), or a custom resource (even an ‘ephemeral’ resource like a search response). But the resource must have a links 
+array that includes an object with a rel  of ‘tiles’ and a type of ‘application/json’, and the `href` must return the Tiles 
+Description object specified below.    
+
+```
+  "links": [
+    ...
+    {
+      "href": "http://data.example.com/collections/buildings/tiles",
+      "rel": "http://www.opengis.net/def/rel/ogc/1.0/tileset"
+      "type": "application/json",
+    }
+  ]
+}
+```
+
+This communicates to any client that the link contains the JSON metadata to access the tiles of this particular resource. 
+
+## TileSet response
+
+The response of this link type is the JSON object that contains the information to enable a client application to formulate a 
+tile request that represents the resources that created the link. The current minimum response in the specification is:
+
+```
+{
+  ...
+  "tileMatrixSetURI": "http://www.opengis.net/def/tilematrixset/OGC/1.0/WorldMercatorWGS84Quad",
```

Comment:
This probably should be WebMercatorQuad in the guide, not WorldMercatorWGS84Quad?

Reply:
Thanks, will update - I get confused by the options and didn't double check, and this I think was the default in the spec? 

...

Yup - https://github.com/opengeospatial/ogcapi-tiles/blob/master/core/standard/clause_7_tile_tileSet.adoc#tileset-resource Could be good to update in the spec too...